audiostack/files/file.py

```diff
@@ -0,0 +1,231 @@
+import os
+import time
+from typing import Optional
+from uuid import UUID
+
+from audiostack import TIMEOUT_THRESHOLD_S
+from audiostack.helpers.request_interface import RequestInterface
+from audiostack.helpers.request_types import RequestTypes
+
+
+class File:
+    """File management class for handling file operations in AudioStack.
+
+    This class provides methods for creating, retrieving, and deleting files
+    in the AudioStack system.
+    """
+
+    FAMILY = "files"
+    interface = RequestInterface(family=FAMILY)
+
+    class Item:
+        """Represents a file item in the AudioStack system.
+
+        This class encapsulates file metadata and provides methods for
+        file operations like downloading.
+        """
+
+        def __init__(self, response: dict) -> None:
+            """Initialize a File.Item instance from API response data.
+
+            Args:
+                response: Dictionary containing file metadata from the API.
+            """
+            self.fileId: str = str(response["fileId"])  # Hard link
+            self.fileName: str = response["fileName"]
+            self.folderId: str = str(response["folderId"])
+            self.url: Optional[str] = response.get("url")
+            self.createdBy: str = response["createdBy"]
+            self.lastModified: Optional[str] = response.get("lastModified")
+            self.fileType: dict = response["fileType"]
+            self.fileCategory: Optional[dict] = response.get("fileCategory")
+            self.size: int = response["size"]
+            self.createdAt: str = response["createdAt"]
+            self.status: str = response["status"]
+            self.duration: Optional[float] = response.get("duration")
+
+        def download(self, fileName: str, path: str = "./") -> None:
+            """Download the file to the specified local path.
+
+            Args:
+                fileName: Name to save the file as locally.
+                path: Directory path where the file should be saved.
+                    Defaults to "./".
+
+            Raises:
+                Exception: If no URL is available for the file.
+            """
+            if not self.url:
+                raise Exception(
+                    "No URL found for this file. Please check the file has been processed."
+                )
+            RequestInterface.download_url(url=self.url, destination=path, name=fileName)
+
+    @staticmethod
+    def get(fileId: str) -> Item:
+        """Retrieve a file by its hard link ID.
+
+        Args:
+            fileId: The unique hard link identifier of the file to retrieve.
```

Comment:
This is confusing for the User. He won't know that we treat a fileId as a hardLinkId. 
Please change to `the unique file id`.